### PR TITLE
Fix rendering of projects and subprojects menu bar

### PR DESCRIPTION
**Pull request by syncguru**
Fixes #21137

screen shot 2016-08-13 at 11 00 15 pm

### CODE REVIEW

File: core/layout_api.php [hunk 1]
```diff
@@ -969,15 +974,13 @@ function layout_main_content_end() {
  */
 function layout_page_content_begin() {
 	echo '  <div class="page-content">' , "\n";
-	echo '    <div class="row">' , "\n";
```
Comment (vboctor) [inline core/layout_api.php]: This has no impact on other pages?

File: core/html_api.php [hunk 2]
```diff
@@ -423,44 +423,43 @@ function print_project_menu_bar() {
...
 	foreach( $t_subprojects as $t_subproject ) {
-		echo $t_char . ' <a href="' . helper_mantis_url( 'set_project.php?project_id=' . $p_parents . $t_subproject ) . '">' . string_html_specialchars( project_get_field( $t_subproject, 'name' ) ) . '</a>';
-		print_subproject_menu_bar( $t_subproject, $p_parents . $t_subproject . ';' );
-		$t_char = ',';
+		$t_active = $t_current_project_id == $t_subproject ? 'active' : '';
+		echo '<a class="btn btn-xs btn-white btn-default ' . $t_active .
+			'" href="' . helper_mantis_url( 'set_project.php?project_id=' . $p_parents . $t_subproject ) . '"><i class="ace-icon fa fa-angle-double-right"></i> ' .
+			string_html_specialchars( project_get_field( $t_subproject, 'name' ) ) . '</a>';
+
+		print_subproject_menu_bar( $t_current_project_id, $t_subproject, $p_parents . $t_subproject . ';' );
```
Comment (vboctor) [inline core/html_api.php]: Are these ids swapped?

**Comment by vboctor** [comment]
:+1:  with a couple of comments.

**Comment by dregad** [comment]
+1, didn't test if there is any regression per Victor's comment above.

File: core/layout_api.php (hunk 1 above)
Comment (syncguru) [inline core/layout_api.php]: I moved the markup to the calling function (one level up) it should not have any side effect.

**Comment by syncguru** [comment]
@vboctor, @dregad comments addressed.

**Comment by dregad** [comment]
Merged in e1eb08b12aa9c91570771cc48f9c47e7b3c3cce8

**Pull request closed without merging**